.github/workflows/witness.yml

```diff
@@ -48,5 +48,9 @@ jobs:
               step: ${{ inputs.step }}
               attestations: ${{ inputs.attestations }}
               command: /bin/sh -c "${{ inputs.command }}"
+
           - if: ${{ inputs.pull_request == true }}
             run: ${{ inputs.command }}
+
+          - if: ${{ inputs.step == 'tests' }}
```

Comment:
This is good for now. Eventually, we've talked about moving this workflow to an external location (i.e. witness GHA repo). We may want a generic "post" step for the job. The `go-witness` [repo](https://github.com/in-toto/go-witness/blob/main/.github/workflows/witness.yml) has an upload artifact step today as a "post" step.

Reply:
I was struggling with it while designing how to add the upload to codecov.
I couldn't [retrieve it from the artifact](https://github.com/in-toto/archivista/actions/runs/7408375375/job/20156574012).

Another alternative is to use the `codecov` client ( (it is documented by codecov), but I don't like this curl approach. So, I could add it to the `pipeline.yml` right after the `make tests`.

```
curl -Os https://uploader.codecov.io/latest/linux/codecov

chmod +x codecov
./codecov
```

Reply:
I opened the issue #146 for it.